hello-strongbox-gradle/build.gradle

```diff
@@ -35,7 +35,7 @@ publishing {
 
             groupId = "org.carlspring.strongbox.examples"
             artifactId = "hello-strongbox-gradle"
-            version = "1.0-SNAPSHOT"
+            version = "1.0-SNAPSHOT" + System.getenv("PACKAGE_REVISION") ?: ""
```

Comment:
This is actually incorrect and will cause more confustion.
* The Maven snapshot format looks like this:
```
1.0-20201030.181125-1
```
This translates to `1.0-SNAPSHOT`, if it's mapped by Maven correctly in you `maven-metadata.xml` file. If you put the date like this, it will consider the date part a `classifier`.
* My understanding is that the Github Package Registry has no understanding of the concept of Maven snapshots.
* We would like users of the example projects to not be overly confused as to what's going on and why we're doing this wizardry. We should add comments in the code where this is specifically for the deployment to the Github Package Registry.

The more I think of it, the more it makes sense to try to override these versions during run time, in order to avoid confusing people as explained above. We can do this for the Maven-based tools easily and (only) when deploying to GPR, we should set a version like `20201030.181125`. This is not perfect, I know, but it's more human-understandable than using the hashes, or we could use `20201030.${shortSHA}`.

Reply:
Okay, I've pushed updates.